include/openthread/tcat.h

```diff
@@ -156,27 +166,30 @@ typedef struct otTcatVendorInfo
     const char                     *mPskdString;          ///< Vendor managed pre-shared key for device
     const char                     *mInstallCode;         ///< Vendor managed install code string
     const otTcatAdvertisedDeviceId *mAdvertisedDeviceIds; /** Vendor managed advertised device ID array.
-                                         Array is terminated like C string with OT_TCAT_DEVICE_ID_EMPTY */
+                                        Array is terminated like C string with OT_TCAT_DEVICE_ID_EMPTY */
     const otTcatGeneralDeviceId *mGeneralDeviceId;        /** Vendor managed general device ID array.
-                                               (if NULL: device ID is set to EUI-64 in binary format)*/
+                                        (if NULL: device ID is set to EUI-64 in binary format) */
+    const char *mApplicationServiceName[OT_TCAT_APPLICATION_LAYER_MAX_COUNT]; /** Array with
+                   application service names as C string with maximum length
+                   OT_TCAT_SERVICE_NAME_MAX_LENGTH or NULL if not supported */
+    bool mApplicationServiceIsTcp[OT_TCAT_APPLICATION_LAYER_MAX_COUNT];       /** Array with
+                         boolean values indicating if the service is of TCP type (otherwise UDP) */
 
 } otTcatVendorInfo;
 
 /**
- * Pointer to call when application data was received over a TCAT TLS connection.
+ * Pointer to call when vendor specific data was received over a TCAT TLS connection.
```

Comment:
The data type name has "Application data" in it. As I understand it supports both the application-layer data, as well as the vendor-specific data (which is really just like another app-layer except it doesn't have a formal name defined).

So in keeping with the rest of the naming, maybe "application data" would be better? Or we say:

```suggestion
 * Pointer to call when application data or vendor-specific data was received over a TCAT TLS connection.
```

Reply:
Fixed.
I also copied some additional text from otHandleBleSecureReceive(...) here to describe that TCAT agent will respond with OT_TCAT_STATUS_UNSUPPORTED if no response is created inside the callback.